services/scraper/handler.go

```diff
@@ -0,0 +1,26 @@
+package scraper
+
+import (
+	"fmt"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocolly/colly/v2"
```

Comment:
For me, I think it would be better if we write our own scraper without using any full featured library/framework so we can learn under the hood in the IC's. You can go with colly, I just shared my thought 😄

Reply:
Yeah you can do that 🍀

Reply:
@junan The IC is not really meant to be a research project. Picking and using the right libraries is part of making this exercise practical. Just like in a client project, you would not re-implement an existing tool, an IC project gas the same requirements.